frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -393,18 +393,27 @@ export class RotkehlchenApi {
 
   queryTaskResult<T>(
     id: number,
-    numericKeys?: string[] | null
+    numericKeys?: string[] | null,
```

Comment:
At some point we also need to finish migrating to zod schemas so that we can get rid of the `numeriKeys` thing.